bin/fin

```diff
@@ -4826,6 +4828,19 @@ case "$1" in
 		shift
 		restart "$@"
 		;;
+	reset)
+		shift
+		# Make sure docker daemon is accessible
+		check_docker_running
```

Comment:
Why is this necessary specifically here, but not with other subcommands?

Reply:
It actually should be here for every command because commands should not do it themselves so that we could call one commands from another without re-checking docker and throwing unneeded errors. But I didn't want to re-write half of fin right now

Reply:
@david-hernandez do you have a recommendation as a native speaker? :)

Reply:
@lmakarov I dont understand the question. Did you mean to put this here, or the comment above about shortcut/shorthand?